apps/ideaspace/src/components/common/IdeaCard/IdeaCard.js

```diff
@@ -48,52 +54,62 @@ function IdeaCard({ cards, cardType }) {
       );
 
       if (res.status === 200) {
-        setTagContent("workshopping");
+        setTagContent('workshopping');
       }
     } catch (error) {
       confirmFailure();
       setButtonContent(`REACTIVATE THIS IDEA`);
     }
-  }
+  };
 
   return (
-    <atoms.Box flexDirection='column'
-      style={{ border: "0.05rem solid rgba(240, 237, 238, 1)", borderRadius: "1rem" }}
+    <atoms.Box
+      flexDirection="column"
+      style={{
+        border: '0.05rem solid rgba(240, 237, 238, 1)',
+        borderRadius: '1rem',
+      }}
     >
-
       <atoms.Box>
-        <IdeaCardTag
-          status={tagContent}
-        />
+        <IdeaCardTag status={tagContent} />
       </atoms.Box>
 
-      <IdeaCardImg
-        cardId={cards.id}
-        cardImg={cards.imgSrc}
-      />
+      <IdeaCardImg cardId={cards.id} cardImg={cards.imgSrc} />
 
       <Link href={{ pathname: urlPath }}>
-        <atoms.Box flexDirection='column' alignItems='flex-start' justifyContent='space-between'
-          padding='0rem 2rem 2rem' style={{ maxWidth: '18.5rem' }}>
-
-          <atoms.Typography type='h3' style={{ fontSize: '1.5rem', marginBottom: '2rem' }}>
+        <atoms.Box
+          flexDirection="column"
+          alignItems="flex-start"
+          justifyContent="space-between"
+          padding="0rem 2rem 2rem"
+          style={{ maxWidth: '18.5rem' }}
+        >
+          <atoms.Typography
+            type="h3"
+            style={{ fontSize: '1.5rem', marginBottom: '2rem' }}
+          >
             {cards.ideaName}
           </atoms.Typography>
 
-          <IdeaCardComment
-            commentLength={cards.comments.length}
-          />
+          <IdeaCardComment commentLength={cards.comments.length} />
 
-          <IdeaCardUpdated
-            updatedAt={cards.updated_at}
-          />
+          <IdeaCardUpdated updatedAt={cards.updated_at} />
         </atoms.Box>
       </Link>
 
-      {tagContent == "archived" ? (
+      <atoms.Box padding="0rem 2rem 2rem">
+        <LikeButton
+          onClick={() => setLiked((prev) => !prev)}
+          filled={liked}
+          text={liked ? 1 : ''}
+          disabled={useUserDataContext().isAuthenticated ? false : true}
```

Comment:
Can we import `isAuthenticated` on line 20 of this file? We are already calling `useUserDataContext()` there. 

This would make the code slightly (not anything noticeable more performant) and easier to read.